test/e2e/build-indicator/test/index.test.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-env jest */
+import { join } from 'path'
+import { retry } from 'next-test-utils'
+import { nextTestSetup, isNextDev, isNextStart } from 'e2e-utils'
+
+const installCheckVisible = (browser) => {
+  return browser.eval(`(function() {
+      window.checkInterval = setInterval(function() {
+      const root = document.querySelector('nextjs-portal').shadowRoot;
+      const indicator = root.querySelector('[data-next-mark]')
+      if(!indicator) return
+      window.showedBuilder = window.showedBuilder || (
+        indicator.getAttribute('data-next-mark-loading') === 'true'
+      )
+      if (window.showedBuilder) clearInterval(window.checkInterval)
+    }, 5)
+  })()`)
+}
+
+describe('Build Activity Indicator', () => {
+  // Use describe.skip so that this suite does not fail with "no tests" during deploy tests.
+  ;(isNextStart ? describe : describe.skip)('Invalid position config', () => {
```

Comment:
When #83602 is merged, we should be able to test `next dev` as well.

Reply:
https://github.com/vercel/next.js/pull/83602/commits/6cdade67e9f0cfa1b4c3b664cdaabb1e5e681a7a